students/byungmoon/westargram/requirements.txt

```
@@ -0,0 +1,5 @@
+
+Django==3.2.4
+django-cors-headers==3.7.0
+
+PyMySQL==1.0.2
```

병문님 해당 파일의 빈 라인은 전부 지워주세요~

```suggestion
Django==3.2.4
django-cors-headers==3.7.0
PyMySQL==1.0.2
```